client/src/app/components/manage-inventory-items/manage-inventory-items.component.ts

```diff
@@ -1,20 +1,39 @@
-import { Component, OnInit } from '@angular/core';
-import { ManageInventoryItemsService } from 'src/app/services/manage-inventory-items.service';
-import { MatPaginator } from '@angular/material/paginator';
-import { MatTableDataSource } from '@angular/material/table';
-import { MatSort } from '@angular/material/sort';
-import { ViewChild } from '@angular/core';
+import {Component, OnInit} from '@angular/core';
+import {ViewChild} from '@angular/core';
+
+import {ManageInventoryItemsService} from 'src/app/services/manage-inventory-items.service';
+
+import {MatPaginator} from '@angular/material/paginator';
+import {PageEvent} from '@angular/material/paginator';
+
+import {MatTableDataSource} from '@angular/material/table';
+import {MatSort} from '@angular/material/sort';
+
```

Comment:
Why remove the spaces and reformat it like this?

Reply:
Those were auto reformatted by the IDE